examples/vault-cluster-enterprise/README.md

```diff
@@ -0,0 +1,73 @@
+Enterprise versions installed, which you can do using the [vault-consul-image example](https://github.com/hashicorp/terraform-google-vault/tree/master/examples/vault-consul-image)). Keep in mind, you must specify
+the Vault & Consul Enterprise download URLs using the `VAULT_DOWNLOAD_URL` and `CONSUL_DOWNLOAD_URL` environment variables respectively.
+
+Note that a Google Load Balancer requires a Health Check to confirm that the Vault nodes are healthy, but at this time,
+Google Cloud only supports [associating HTTP Health Checks with a Target Pool](
+https://github.com/terraform-providers/terraform-provider-google/issues/18), not HTTPS Health Checks. The recommended
+workaround is to run a separate proxy server that listens over HTTP and forwards requests to the HTTPS Vault endpoint.
+We accomplish this by using the [run-nginx](https://github.com/hashicorp/terraform-google-vault/tree/master/modules/run-nginx) module to run the web server. 
+
+For more info on how the Vault cluster works, check out the [vault-cluster](https://github.com/hashicorp/terraform-google-vault/tree/master/modules/vault-cluster) documentation.
+
+**Note:** This example will automatically create a Google Cloud KMS key. You can disable this behaviour by setting the `var.create_kms_crypto_key` variable to false. Crypto Keys cannot be deleted from Google Cloud Platform, however their versions can. Terraform will by default, erase all
+Crypto Key versions when destroying the resource making any data encrypted by the key unrecoverable. For this reason we recommend reusing an
+existing Cloud KMS key in production.
+
+## Quick start
+
+To deploy a Vault Cluster:
+
+1. `git clone` this repo to your computer.
+1. Build a Vault and Consul Enterprise Google Image. See the [vault-consul-image example](https://github.com/hashicorp/terraform-google-vault/tree/master/examples/vault-consul-image) documentation
+   for instructions. Make sure to note down the ID of the Google Image.
+1. Install [Terraform](https://www.terraform.io/).
+1. Make sure your local environment is authenticated to Google Cloud.
+1. Open `variables.tf` and fill in any variables that don't have a default, including putting your Google Image ID into
+   the `vault_source_image` and `consul_server_source_image` variables. Alternatively, initialize the variables by creating
+   a `terraform.tfvars` file.
+1. Add your Vault Enterprise license key to the directory. See: `vault-license.hclic`. **Note: That in production we recommend encrypting this key. See below for more information.**
+1. Run `terraform init`.
+1. Run `terraform plan`.
+1. If the plan looks good, run `terraform apply`.
+1. To enable other Compute Instances in the same GCP Project to access the Vault Cluster, edit the `main.tf` file to 
+   modify the `allowed_inbound_tags_api` variables. To allow arbitary IP addresses to access the Vault cluster from
+   within the VPC, modify the `allowed_inbound_cidr_blocks_api` variable.
```

Comment:
I actually just think this item shouldn't just be here. Since it'd the main.tf of the example, not the cluster, I'd rather we just allowed `0.0.0.0/0` for test purposes, explaining what this value does, with a big warning that this is just for the example and shouldn't be on production. I actually think making this an additional variable obscures the understanding of the example